clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleHttpLoadBalancerAtomicOperation.groovy

```diff
@@ -105,7 +105,7 @@ class DeleteGoogleHttpLoadBalancerAtomicOperation extends DeleteGoogleLoadBalanc
     List<String> listenersToDelete = []
     projectForwardingRules.each { ForwardingRule rule ->
       def proxy = GCEUtil.getTargetProxyFromRule(compute, project, rule)
-      if (GCEUtil.getLocalName(proxy?.getUrlMap()) == urlMapName) {
+      if (proxy?.urlMap && GCEUtil.getLocalName(proxy?.urlMap) == urlMapName) {
```

Comment:
Actually I think this was NPE'ing from `getUrlMap` and the first check is extraneous. I'll remove it and test that it's working fine.